router/batchrouter/asyncdestinationmanager/bing-ads/audience/bingads_suite_test.go

```diff
@@ -1,4 +1,4 @@
-package bingads
+package bingads_audience
```

Comment:
```suggestion
package audience
```
Per convention is ok to name this just `audience`, name conflicts can be solved on the package that imports it.

Reply:
changed